trin-core/src/cli.rs

```diff
@@ -72,6 +72,14 @@ pub struct TrinConfig {
         help = "Hex encoded 32 byte private key (considered unsafe to pass in pk as cli arg, as it's stored in terminal history - keyfile support coming soon)"
     )]
     pub private_key: Option<HexData>,
+
+    #[structopt(
+        long = "networks",
+        help = "Comma-separated list of which portal networks to activate",
+        default_value = "history,state",
```

Comment:
I'm not sure whether it would be better to instead use activation flags for each desired network. Of course this creates more CLI arguments, and committing to this pattern would require a new CLI argument for each subnetwork. However, it aligns with the mental model I have for a portal client, where the user may toggle each subnetwork on and off, and it simplifies the code that interprets the values for these arguments.

Reply:
> it would be better to instead use activation flags for each desired network

I think we can address this in a separate issue if it seems important to you.

Reply:
Yeah, this is an interesting question. Imo it's somewhat out of scope for this pr, since it's a relatively simple change that can be implemented as various use cases arise and justify it. I created an issue [here](https://github.com/ethereum/trin/issues/90) to track this conversation - and just going with `DEFAULT_NETWORKS` for now.